tester/orb.yaml

```diff
@@ -2,6 +2,66 @@ version: 2.1
 description: "Tools for running test commands."
 
 jobs:
+  pipcheck:
```

Comment:
What existing/recent cases would this check help us avoid?

Reply:
So `pip check` throws an error whenever you have multiple dependencies installed which conflict with each other. Eg. if you have the following two requirements files:
```
# myapp/requirements.txt
ujson==1.34
mylib==1.0.0
```
and
```
# mylib/requirements.txt
ujson==1.35
```

In the above case, you'll non-deterministically get a different `ujson` version with each installation (pip's dependency resolver ain't very fancy). `pip check` will warn you that those libraries aren't compatible since they have differing trasient dependencies on `ujson`.

Specific to us, I'm hoping this will help fix some of the annoyances around `voiceai-common` versions not being quite what we expect depending on installation order of NLP packages.